abilities/music/youtube.py

```diff
@@ -1,3 +1,4 @@
+import asyncio
```

Comment:
Now that you are using a PEP 604 union (`self.peeked_packet: bytes | None = None`), it is necessary to add `from __future__ import annotations` as an import at the top of the file to ensure reverse compatibility.
```suggestion
from __future__ import annotations

import asyncio
```